eloquent-resources.md

```diff
@@ -126,6 +126,8 @@ After defining your resource collection, it may be returned from a route or cont
         return new UserCollection(User::all());
     });
 
+> {note} Collections will automatically utilise any singular resources defined (a resource class matching the start of the collection class name, e.g.: `JokeCollection` will try to match `Joke`.) If you wish to define what singular resource should be used instead of this default behaviour you can define `$this->collects` from within your collection class.
```

Comment:
I believe we use American English here, so 'utilise' --> 'utilize'.

Reply:
Sure thing, I'll swap out the Ozzy for American.